docs/examples/DataSet/Dataset Context Manager.ipynb

```diff
@@ -131,10 +140,10 @@
     "\n",
     "Furthermore, datasets are associated to an experiment. By default the run\n",
     "is appended to the latest existing experiments. If no experiment has been created,\n",
-    "we must create one. We do that by calling `load_or_create_experiment`\n",
+    "we must create one. We do that by calling `load_or_create_experiment` function.\n",
```

Comment:
I preferred the previous version. It is irrelevant whether `load_or_create_experiment` is a function or any other kind of `Callable`.

Reply:
From software development point of view, yes, I agree with you. But this is a document? We are not supposed to follow some coherency in the English? I can not, personally, treat documents as code. It should be readable.

Reply:
@Dominik-Vogel i think that in this particular case the difference is subtle. However, for other cases I think it's good to mention whether a "symbol" is a function or some method of some class because this helps the beginner reader to find where the symbol is.

Reply:
I am fine with this. (We can add a 'the')